Parsers/AS-McAfeeSolidcore/AS-McAfeeSolidcore.yaml

```diff
@@ -0,0 +1,21 @@
+Function:
```

Comment:
Hi @AcceleryntSecurityDev, the `id` is missing, can you please add that?

Also can I please know why have you kept the txt, any particular reason, if not, please remove it, thanks

Reply:
@v-rbajaj what value should I populate the id with? I will remove the txt file now. Thanks

Reply:
You need to add GUID - you can generate the GUID from here - https://guidgenerator.com/